src/debugtopus.ts

```diff
@@ -111,27 +78,35 @@ export const prepareTestRun = async ({
 
   const outputDir = path.join(tempDir, "output");
 
-  const testFilePaths: string[] = [];
-
-  for (const testCase of testCasesWithCode) {
-    const testFilePath = getUniqueFilename(tempDir, testCase);
-    writeFileSync(testFilePath, testCase.code);
-    testFilePaths.push(testFilePath);
-  }
-
   const fileNameUUID = randomUUID();
   const configFilePath = path.join(tempDir, `${fileNameUUID}.config.ts`);
-  writeFileSync(configFilePath, getConfig(url, outputDir, basicAuth));
-
   return {
-    testFilePaths,
+    outputDir,
     configFilePath,
     testDirectory: tempDir,
-    outputDir,
     packageRootDir,
+    testFilePaths: [],
```

Comment:
ensure by typing that this is NOT yet returned here, otherwise someone could still think it's populated.